packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.controller.js

```diff
@@ -0,0 +1,51 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from './loadbalancers.constants';
+
+export default class OctaviaLoadBalancersCtrl {
+  /* @ngInject */
+  constructor(atInternet, Alerter, $translate) {
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  $postLink() {
+    if (this.loadBalancerDeleteSuccess) {
```

Comment:
this can be done in delete component so you can remove the deleteSuccess params from state

Reply:
Done